### ISSUE TITLE
Filter on DoE for OwnedThing

### ISSUE THREAD

**Issue by naronatrhea** [RHEAGROUP/CDP4-IME-Community-Edition#62]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the CDP4-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
a filter on DoE shall be added on OwnedThing so that a user without the proper credential cannot just select whichever domain

Solution:
- Create a reusable component that can do:
   - In all browsers and dialogs of OwnedThings implement this filter.
   - Visually show the current DoE in bold
   - On creation (dialog) add current DoE as default.

### Logs

  - [ ] There are no logs regarding the issue
  - { ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- CDP4 version:
  - [ ] CDP4-IME: 1.0.17
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4 Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

**Comment by samatrhea** [comment]
good idea, i also suggest that the owner field for a new Thing shall be set to the DomainOfExpertise of the current session

**Comment by MartinFijneman** [comment]
verified. seems to have been implemented, DoE for current session filled in.

tested with:
CDP4-COMET version: 11.1.0 rc2
